glibc.yaml

```diff
@@ -181,6 +185,12 @@ pipeline:
       mkdir -p ${{targets.destdir}}/etc/apk/commit_hooks.d
       cp ldconfig-commit.sh -p ${{targets.destdir}}/etc/apk/commit_hooks.d/
 
+  - name: "Clean up /etc/ld.so.cache"
+    runs: |
+      # Having a package own a file that will be different in most
+      # every image poisons our layering mechanism
+      rm ${{targets.destdir}}/etc/ld.so.cache
+
```

Comment:
I thought this got done in a different pr already.

Reply:
There was a draft PR that @dannf was working on, but we decided to join efforts and make a single PR for everything.